fullscreen.bs

```diff
@@ -396,6 +396,15 @@ could be an open <{dialog}> element.
  <a>simple fullscreen document</a>, then set <var>doc</var> to <var>topLevelDoc</var> and
  <var>resize</var> to true.
 
+ <li><p>If <var>doc</var>'s <a>fullscreen element</a> is not connected:
+  <ol>
+   <li><p><a for=set>Append</a> (<code>fullscreenchange</code>, <var>doc</var>'s
```

Comment:
I tried firing it in a microtask and it seemed to work fine. However I kept it the same as all the other events. Because in the special cases (like current Step 10.1 in requesting fullscreen when an error occurs it still queues the event for at rAF time).